extensions/amp-list/amp-list.md

```diff
@@ -294,6 +294,22 @@ The `items` array will be truncated to `max-items` entries if the returned value
 Causes `<amp-list>` to treat the returned result as if it were a single element array. An object response will be wrapped in an array so
 `{items: {...}}` will behave as if it were `{items: [{...}]}`.
 
+##### strip-prefix (optional)
+
+Causes `<amp-list>` to strip a prefix from the fetched JSON before parsing. This can be useful for APIs that include [security prefixes](http://patorjk.com/blog/2013/02/05/crafty-tricks-for-avoiding-xssi/) like `)]}` to help prevent cross site scripting attacks.
+
+For example, lets say we had an API that returned this response:
+
+```
+)]}{ 'a': 'value' }
```

Comment:
Let's make this example data shape similar to the JSON that `amp-list` typically expects to receive:
`{ "items": [...] }`

Reply:
Done, PTAL